pkgs/by-name/do/docbook/generic-xml-dtd.nix

```diff
@@ -0,0 +1,61 @@
+{
+  version,
+  url ? null,
+  hash,
+  preInstall ? "",
```

Comment:
Personally I prefer not to declare phases as inputs. `override`  is better suited for this.